in/command_test.go

```diff
@@ -79,6 +79,17 @@ var _ = Describe("In Command", func() {
 			})
 		})
 
+		Context("when configured to skip download", func() {
+			BeforeEach(func() {
+				request.Source.SkipDownload = true
+			})
+
+			It("downloads the existing version of the file", func() {
```

Comment:
This says the opposite of what you want to demonstrate, and doesn't really assert anything. Could you reword this and add assertions that it doesn't download the file?

Something like `Expect(s3client.DownloadFileCallCount()).To(Equal(0))`.